main/lsp/tools/make_lsp_types.cc

```diff
@@ -1373,6 +1374,7 @@ void makeLSPTypes(vector<shared_ptr<JSONClassType>> &enumTypes, vector<shared_pt
                                                 {"workspace/symbol", WorkspaceSymbolParams},
```

Comment:
Can you also add `sorbetShowSymbolProvider` as an optional boolean to `ServerCapabilities`, and then update the initialize request to set it?

https://github.com/sorbet/sorbet/blob/40e3746f/main/lsp/requests/initialize.cc#L18

That will enable the client to conditionally surface the option to copy reference only if the server supports it.